duktape.js

```diff
@@ -139,3 +144,44 @@ fs.readdirSync('.').forEach(function (filename) {
 
 console.log(testCount + ' tests executed: ' + testSuccess + ' success, ' + (testCount - testSuccess) + ' fail');
 console.log(testOutOfDate + ' tests are out of date (data-*.js file .res)');
+
+console.log("Writing New Results To File...");
+for (var i = 0; i < fixTests.length; i++) {
+    var path = fixTests[i].path;
+    var success = fixTests[i].success;
+    var data = fs.readFileSync(path[0] + ".js", "utf8").split("\n");
+    var k = 1;
+    var done = false;
+    var line = 0;
+    while (!done) {
+        if (data[line].includes("'" + path[k].replace(new RegExp("'", "g"), "\\'") + "'")) {
+            k++;
+        }
+        if (k >= path.length) {
+            done = true;
+        } else {
+            line++;
+        }
+    }
+    while (!data[line].includes("res: {")) {
+        line++;
+    }
+    var keyExists = false;
+    while (!data[line].includes("}")) {
```

Comment:
This algorithm seems to be highly coupled to unenforced covnentions - which seems brittle.